src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -797,7 +1159,8 @@ void QFieldCloudProjectsModel::uploadProject( const QString &projectId, const bo
 
   if ( shouldDownloadUpdates && deltaFileWrapper->count() == 0 )
   {
-    downloadProject( projectId );
+    project->status = ProjectStatus::Idle;
```

Comment:
Hijacking the line here to comment about the function name -- you've renamed other functions to make sure that project was the function prefix; in this case it'd be projectUpload vs. uploadProject. Do you want to rename that here? 

That'll make it easier when we eventually de-couple the model from individual project class post 1.0.

Reply:
Done. I left it to reduce the confusion why I do it in this PR, but if you ask, I don't have any other objections. Having custom `CloudProject` class is more than needed, can't wait having this in place, will save a lot of time and errors (e.g. all these checks if the project is valid above).

Reply:
I'd say go for the renaming in this PR since you did so for other functions. It'll help us down the road.